Data/ByteString.hs

```diff
@@ -44,169 +44,169 @@ module Data.ByteString (
         ByteString,             -- abstract, instances: Eq, Ord, Show, Read, Data, Typeable, Monoid
```

Comment:
Can you also remove these comments here and in the other modules?

```suggestion
        ByteString,
```

Reply:
On my way!